template.toml

```diff
@@ -127,6 +127,7 @@ parse_deps = false
 # include = []
 exclude = []
 clean = false
+# deps_with_top_level_items = []
```

Comment:
```suggestion
deps_with_top_level_items = []
```

I believe this should be uncommented since the default is really the same as `[]`